conf/application-json-logger.xml

```diff
@@ -7,7 +7,13 @@
 
     <logger name="uk.gov" level="${logger.uk.gov:-INFO}"/>
 
-    <logger name="application" level="${logger.application:-INFO}"/>
+    <!--  uses the default "application" logger level: http://logback.qos.ch/manual/configuration.html#defaultValuesForVariables  -->
+    <logger name="application"  level="${logger.application:-INFO}"/>
+    <logger name="connectors"   level="${logger.application:-INFO}"/>
+    <logger name="controllers"  level="${logger.application:-INFO}"/>
+    <logger name="model"        level="${logger.application:-INFO}"/>
```

Comment:
model and utils are not required because we are not logging anything in these packages